haiku-apps/bescreencapture/bescreencapture-2.1.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="A screen recorder utility"
+DESCRIPTION="BeScreenCapture is a screen recorder utility for Haiku.
+It allows you to record what happens on your screen, then save it \
+to any media format supported in Haiku.
+BeScreenCapture can record either the entire screen, or just a section you \
+select."
+
+SUMMARY_inputfilter="Shortcut handler for BeScreenCapture"
+DESCRIPTION_inputfilter="Input Server Addon for BeScreenCapture. Allows the \
+user to launch BeScreenCapture and start/stop recording using a keyboard \
+combination (CTRL-COMMAND-SHIFT + R)."
+
+HOMEPAGE="https://github.com/jackburton79/bescreencapture"
+SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v2.1.tar.gz"
+CHECKSUM_SHA256="5b3ff575fcfe7519556fcd3d9487ef5491ead656f1cffd79b78dfcb3a52332f8"
+SOURCE_FILENAME="bescreencapture-2.1.tar.gz"
+LICENSE="
+	BSD (3-clause)
+	MIT
+	"
+COPYRIGHT="2014-2016 Stefano Ceccherini"
+REVISION="1"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	bescreencapture = $portVersion
+	app:BeScreenCapture = $portVersion
+	"
+
+PROVIDES_inputfilter="
+	bescreencapture_inputfilter = $portVersion
+	app:BeScreenCaptureInputFilter = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	"
+REQUIRES_inputfilter="
+	haiku
+	bescreencapture == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:gcc
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects \
+		BUILDHOME=`finddir B_SYSTEM_DEVELOP_DIRECTORY`
+
+	make -C inputfilter OBJ_DIR=objects \
+		BUILDHOME=`finddir B_SYSTEM_DEVELOP_DIRECTORY`
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp -a objects/BeScreenCapture $appsDir
+	mkdir -p $addOnsDir/input_server/filters
+	cp -a inputfilter/objects/BeScreenCaptureInputFilter $addOnsDir/input_server/filters
+
+	packageEntries inputfilter \
+	add-ons/input_server/filters/BeScreenCaptureInputFilter
```

Comment:
The indentation of the continuation line would look nicer with an extra tab.

BTW, it might also be a good idea to find which environment variable points to the add-ons dir and use it.
To find the name of that variable you can:
```Shell
grep add-ons /path/to/haikuports/haiku-apps/bescreencapture/work-2.1/wrapper-script
```
(I don't remeber what it is and I'm not running Haiku at the moment.)